src/cli/test/reporters/github/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Context, Reporter } from '@jest/reporters';
+import { AggregatedResult, TestResult } from '@jest/test-result';
```

Comment:
This may make it clearer that we're not taking a runtime dependency on these subpackages:

```suggestion
import type { Context, Reporter } from '@jest/reporters';
import type { AggregatedResult, TestResult } from '@jest/test-result';
```